pom.xml

```diff
@@ -117,7 +116,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.8.8</version>
+                <version>0.8.10</version>
```

Comment:
The latest version is [0.8.11](https://mvnrepository.com/artifact/org.jacoco/jacoco-maven-plugin)